frontend/ui/data/koptoptions.lua

```diff
@@ -123,14 +124,14 @@ In 'semi-auto' and 'manual' modes, you may need to define areas once on an odd p
                 event = "DefineZoom",
                 args = {"full", "width", "height"},
                 name_text_hold_callback = optionsutil.showValues,
-                help_text = _([[Set zoom mode.]]),
+                help_text = _([[Set what to fit.]]),
             },
             {
                 name = "zoom_range_number",
                 name_text = "Number",
-                buttonprogress = true,
-                fine_tune = true,
-                fine_tune_param = {value_step = 0.1},
+                toggle =  {_("1"), _("2"), _("3"), _("4"), _("5"), _("6"), _("7"), _("8")},
+                more_options = true,
+                more_options_param = {value_step = 0.1, precision = "%.1f",},
```

Comment:
Setting 2.4 columns or 3.7 lines works / can be useful ?

Reply:
I think so: sometimes, scanned PDF can be very weird (they weren't always treated by [ScanTailor](https://github.com/scantailor/scantailor), and, for example, you might have footnotes that don't take a full row), so there could be corner cases when someone wants to set it, while benefiting from the automatic calculation of overlap.